From 5763c710ef0f8be240e255c786aceea5b32a0dd5 Mon Sep 17 00:00:00 2001
From: Brian Silverman <bsilverman@iveia.com>
Date: Thu, 23 Apr 2020 13:08:11 -0400
Subject: [PATCH] iveia version banner

---
 init/main.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/init/main.c b/init/main.c
index 457eb16..84efdfb 100644
--- a/init/main.c
+++ b/init/main.c
@@ -92,6 +92,7 @@
 #include <linux/rodata_test.h>
 #include <linux/jump_label.h>
 #include <linux/mem_encrypt.h>
+#include <iveia_version.h>
 
 #include <asm/io.h>
 #include <asm/bugs.h>
@@ -552,6 +553,8 @@ asmlinkage __visible void __init start_kernel(void)
 	boot_cpu_init();
 	page_address_init();
 	pr_notice("%s", linux_banner);
+	pr_notice("Src commit:  %s\n", IVEIA_SRC_BUILD_HASH);
+	pr_notice("Meta commit: %s\n", IVEIA_META_BUILD_HASH);
 	setup_arch(&command_line);
 	/*
 	 * Set up the the initial canary and entropy after arch
-- 
2.7.4

