From 91761e0f4c81b6cc36f9b90a98ace0d4aec0f5de Mon Sep 17 00:00:00 2001
From: Brian Silverman <bsilverman@iveia.com>
Date: Thu, 23 Apr 2020 13:08:11 -0400
Subject: [PATCH 1/5] iveia version banner

---
 init/main.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/init/main.c b/init/main.c
index 06b98350ebd2..2ef2a23ec8fe 100644
--- a/init/main.c
+++ b/init/main.c
@@ -101,6 +101,7 @@
 #include <linux/init_syscalls.h>
 #include <linux/stackdepot.h>
 #include <net/net_namespace.h>
+#include <iveia_version.h>
 
 #include <asm/io.h>
 #include <asm/bugs.h>
@@ -954,6 +955,8 @@ asmlinkage __visible void __init __no_sanitize_address start_kernel(void)
 	page_address_init();
 	pr_notice("%s", linux_banner);
 	early_security_init();
+	pr_notice("Src commit:  %s\n", IVEIA_SRC_BUILD_HASH);
+	pr_notice("Meta commit: %s\n", IVEIA_META_BUILD_HASH);
 	setup_arch(&command_line);
 	setup_boot_config();
 	setup_command_line(command_line);
-- 
2.25.1

