diff --git a/drivers/net/mdio/Kconfig b/drivers/net/mdio/Kconfig
index a10cc460d7cf..d43b6635d75e 100644
--- a/drivers/net/mdio/Kconfig
+++ b/drivers/net/mdio/Kconfig
@@ -117,6 +117,16 @@ config MDIO_I2C
 
 	  This is library mode.
 
+config MDIO_I2C_GEN
+	tristate "I2C-based MDIO PHY"
+	depends on OF_MDIO
+	select MDIO_BUS_MUX
+	select MDIO_I2C
+	help
+	  Support for MDIO devices using the MDIO bus in I2C mode (such as
+	  the BSC Slave Mode on the BCM54616S GigE PHY).  Potentially
+	  compatibile with any I2C mode MDIO device.
+
 config MDIO_MVUSB
 	tristate "Marvell USB to MDIO Adapter"
 	depends on USB
diff --git a/drivers/net/mdio/Makefile b/drivers/net/mdio/Makefile
index 5c498dde463f..e15908af0b7a 100644
--- a/drivers/net/mdio/Makefile
+++ b/drivers/net/mdio/Makefile
@@ -11,6 +11,7 @@ obj-$(CONFIG_MDIO_CAVIUM)		+= mdio-cavium.o
 obj-$(CONFIG_MDIO_GPIO)			+= mdio-gpio.o
 obj-$(CONFIG_MDIO_HISI_FEMAC)		+= mdio-hisi-femac.o
 obj-$(CONFIG_MDIO_I2C)			+= mdio-i2c.o
+obj-$(CONFIG_MDIO_I2C_GEN)	+= mdio-i2c-gen.o
 obj-$(CONFIG_MDIO_IPQ4019)		+= mdio-ipq4019.o
 obj-$(CONFIG_MDIO_IPQ8064)		+= mdio-ipq8064.o
 obj-$(CONFIG_MDIO_MOXART)		+= mdio-moxart.o
