From 03d62f018a438c9eddaa9411dfa4d00269857f4a Mon Sep 17 00:00:00 2001
From: Rick Rogers <rrogers@iveia.com>
Date: Tue, 6 Jul 2021 15:24:24 -0400
Subject: [PATCH] force XSDPS_DEFAULT_SPEED_MODE for EMMC boot target

---
 XilinxProcessorIPLib/drivers/sdps/src/xsdps.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/XilinxProcessorIPLib/drivers/sdps/src/xsdps.c b/XilinxProcessorIPLib/drivers/sdps/src/xsdps.c
index bd811a5e7a..44619d893a 100644
--- a/XilinxProcessorIPLib/drivers/sdps/src/xsdps.c
+++ b/XilinxProcessorIPLib/drivers/sdps/src/xsdps.c
@@ -887,6 +887,8 @@ s32 XSdPs_CardInitialize(XSdPs *InstancePtr)
 			InstancePtr->Mode = XSDPS_DEFAULT_SPEED_MODE;
 		}
 
+        InstancePtr->Mode = XSDPS_DEFAULT_SPEED_MODE;
+
 		if (InstancePtr->Mode != XSDPS_DEFAULT_SPEED_MODE) {
 			Status = XSdPs_Change_BusSpeed(InstancePtr);
 			if (Status != XST_SUCCESS) {
