diff -urN build/fsbl/zynqmp_fsbl_bsp/psu_cortexa53_0/include/xparameters.h build.new/fsbl/zynqmp_fsbl_bsp/psu_cortexa53_0/include/xparameters.h
--- build/fsbl/zynqmp_fsbl_bsp/psu_cortexa53_0/include/xparameters.h	2020-03-18 20:55:50.885252691 -0400
+++ build.new/fsbl/zynqmp_fsbl_bsp/psu_cortexa53_0/include/xparameters.h	2020-03-18 20:57:33.101162903 -0400
@@ -27,8 +27,8 @@
 /* Number of Fabric Resets */
 #define XPAR_NUM_FABRIC_RESETS 1
 
-#define STDIN_BASEADDRESS 0xFF000000
-#define STDOUT_BASEADDRESS 0xFF000000
+#define STDIN_BASEADDRESS 0xFF010000
+#define STDOUT_BASEADDRESS 0xFF010000
 
 /******************************************************************/
 
