From 03a93624ee6045219030a2b006c01b7d8118ea66 Mon Sep 17 00:00:00 2001
From: "emil.m" <emil.m@dicortech.com>
Date: Mon, 16 Oct 2023 12:13:28 +0530
Subject: [PATCH] Include ivfru command

---
 cmd/Makefile | 5 +++
 1 file changed, 5 insertions(+)

diff --git a/cmd/Makefile b/cmd/Makefile
index dd86675bf2..a153eee229 100644
--- a/cmd/Makefile
+++ b/cmd/Makefile
@@ -191,6 +191,11 @@ obj-$(CONFIG_CMD_BLOB) += blob.o
 # Android Verified Boot 2.0
 obj-$(CONFIG_CMD_AVB) += avb.o
 
+# iVeia IPMI FRU info utility
+obj-y += ivfru.o
+obj-y += ivfru_common.o
+obj-y += ivfru_plat.o
+
 obj-$(CONFIG_ARM) += arm/
 obj-$(CONFIG_RISCV) += riscv/
 obj-$(CONFIG_X86) += x86/
-- 
2.25.1

