From 705080209e6507fa8df85783d5e6742c5c88043c Mon Sep 17 00:00:00 2001
From: Brian Silverman <bsilverman@iveia.com>
Date: Thu, 26 May 2022 17:04:34 -0400
Subject: [PATCH] include iVeia Kconfig

---
 arch/arm/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index b31b92f5fe..1eabb70462 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -2216,6 +2216,7 @@ source "board/phytium/durian/Kconfig"
 source "board/xen/xenguest_arm64/Kconfig"
 source "board/keymile/Kconfig"
 
+source "board/iveia/Kconfig"
 source "arch/arm/Kconfig.debug"
 
 endmenu
