From 300b0a7e28a8ad89a5625b0e48850dc679162f73 Mon Sep 17 00:00:00 2001
From: Rick Rogers <rrogers@iveia.com>
Date: Tue, 6 Jul 2021 15:24:24 -0400
Subject: [PATCH 2/2] force XSDPS_DEFAULT_SPEED_MODE for EMMC boot target

---
 XilinxProcessorIPLib/drivers/sdps/src/xsdps_card.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/XilinxProcessorIPLib/drivers/sdps/src/xsdps_card.c b/XilinxProcessorIPLib/drivers/sdps/src/xsdps_card.c
index d9335f9ed8..16619a44ae 100644
--- a/XilinxProcessorIPLib/drivers/sdps/src/xsdps_card.c
+++ b/XilinxProcessorIPLib/drivers/sdps/src/xsdps_card.c
@@ -1421,6 +1421,8 @@ void XSdPs_IdentifyEmmcMode(XSdPs *InstancePtr, const u8 *ExtCsd)
 		}
 #endif
 	}
+
+	InstancePtr->Mode = XSDPS_DEFAULT_SPEED_MODE;
 }
 
 /*****************************************************************************/
-- 
2.25.1

